### PR TITLE
ddl: Break campaignLoop when closing DDL

### PR DESCRIPTION
**Pull request by zimulala**
Fix a bug and update logs

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @shenli @tiancaiamao

File: ddl/owner_manager.go [hunk 1]
```diff
@@ -186,16 +188,17 @@ func (m *ownerManager) campaignLoop(ctx goctx.Context, etcdSession *concurrency.
 				time.Duration(ManagerSessionTTL)*time.Second)
 			_, err = m.etcdCli.Revoke(ctx, etcdSession.Lease())
 			cancel()
-			log.Infof("[ddl] break %s campaign loop err %v", key, err)
+			log.Infof("[ddl] %s break campaign loop err %v", idInfo, err)
+			return
```
Comment (shenli) [inline ddl/owner_manager.go]: Why we return here?

Comment (zimulala) [inline ddl/owner_manager.go]: DDL is closed, so we can break this loop.

**Comment by shenli** [comment]
LGTM

**Comment by hanfei1991** [comment]
LGTM

**Pull request merged**